recipes/platform.converters/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.converters"
+    license = "MIT"
+    homepage = "https://github.com/linksplatform/Converters"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """lol"""
```

Comment:
Add as in the following description
https://github.com/conan-io/conan-center-index/blob/5aa68691d7ec486f9c6034a8f199d7eb33f19307/recipes/platform.exceptions/all/conanfile.py#L13-L14